viewer/package.json

```diff
@@ -85,7 +84,10 @@
     "glslify": "^7.0.0",
     "glslify-import": "^3.1.0",
     "lodash": "^4.17.15",
-    "rxjs": "^6.5.5",
+    "rxjs": "^6.5.5"
+  },
+  "peerDependencies": {
+    "@cognite/sdk": "^2.28.0",
     "three": "^0.115.0"
```

Comment:
@larsmoa  Shouldn't we fix it as 0.115.x or do an update in our code? I believe I had some type issues when installed reveal + three 0.116

Reply:
Actually, it's fixed because minor revisions are considered breaking when major version is 0. I.e. `^0.115.0` maps to `0.115.x`.

Reply:
@larsmoa cool. I didn't know that.